vault/acl.go

```diff
@@ -80,8 +145,10 @@ func (a *ACL) Capabilities(path string) (pathCapabilities []string) {
 	// Find an exact matching rule, look for glob if no match
 	var capabilities uint32
 	raw, ok := a.exactRules.Get(path)
+
 	if ok {
-		capabilities = raw.(uint32)
+		perm := raw.(*Permissions)
+		capabilities = perm.CapabilitiesBitmap
```

Comment:
Eventually we'll probably want to change this to add allowed/denied parameters but that can probably come later.